CONTRIBUTING.md

```diff
@@ -5,12 +5,13 @@
 
 2. **Code formatting** rules are described by the [Clang-Format Style Options] file [.clang-format].
    Please use the [clang-format] tool to format your code _changes_ accordingly.
-   
-   Example:
-   
-       clang-format -i -lines=146:159 libevm/JitVM.cpp
+   [git-clang-format] tool is useful to limit reformatting to your changes only:
+
+       git clang-format          # to reformat the changes in the working directory
```

Comment:
It reformats changes in the index, not in the working directory.  Or, more precisely,
it reformats the changes in the index, and puts the formatted version in the working directory.

Reply:
^ hasn't been fixed yet.

Reply:
Thanks for noticing, I didn't check it myself 😞